src/base/optimise/owl_algodiff_generic.ml

```diff
@@ -1083,6 +1083,18 @@ module Make
       A.avg_pool2d_backward p a b s o
       |> pack_arr
 
+    and max_pool3d_backward p a b s o =
```

Comment:
But if you only add these two functions here, how come Algodiff module can know how to differentiate `max_pool3d` and `avg_pool3d` functions?

Reply:
Sorry I thought the forward operations are already included. I'll add them in.